gsa/src/gmp/models/nvt.js

```diff
@@ -113,25 +118,43 @@ const getOtherRefs = refs => {
 class Nvt extends Info {
   static entityType = 'nvt';
 
+  static parseObject(object) {
+    const ret = super.parseObject(object);
+
+    ret.severity = hasValue(ret.score)
+      ? parseScoreToSeverity(ret.score)
+      : undefined;
+
+    if (ret.preferenceCount < 0) {
+      // actually preferenceCount in the XML is -1 in get_info
+      // right now.
+      ret.preferenceCount = ret.preferences.length;
+    }
+
+    return ret;
+  }
+
   static parseElement(element) {
```

Comment:
The parseElement function is used in places where we still get data from gsad with the old XML structure. Therefore, it should not be changed to match Hyperion.